pkg/mockentries.go

```diff
@@ -0,0 +1,124 @@
+package pkg
+
+import (
+	reflect "reflect"
+
+	runtime "github.com/go-openapi/runtime"
+	gomock "github.com/golang/mock/gomock"
+	entries "github.com/sigstore/rekor/pkg/generated/client/entries"
+)
+
+// MockEntriesClientService is a mock of ClientService interface.
+type MockEntriesClientService struct {
```

Comment:
Maybe provide a Makefile target or instructions somewhere on how to regenerate these?